v15-upgrade-log.md

```diff
@@ -38,3 +38,6 @@ Now we will upgrade to SpatialOS 15.0.0.
    the two different units of authority in our project. We could replace the
    EntityAcl with the AuthorityDelegation component, but with the new user-space
    load balancing, we won't need to update authority dynamically.
+1. Update the component registry for the C++ physics worker. We use the
+   `worker::Schema` template to combine components and component sets. We need
+   both, as component sets are now the unit of authority.
```

Comment:
Does this mean you *have* pass the component sets to `worker::Schema` now or could you have passed the components belonging to those component sets individually and it would still work?

Reply:
You have to pass both, so that your view can correctly be updated when you receive a `ComponentSetAuthorityChangeOp`. This is because the op does not contain any information about the components which are in the component set, but do not exist on the entity, and an interaction with dynamic components.